### ISSUE TITLE
Stop talking to NickServ

### ISSUE THREAD

**Issue by AlexDaniel** [perl6/whateverable#213]
```
▬▬▶ _ :NickServ!NickServ@services. NOTICE evalable6test You are now identified for whateverable.
◀▬▬ _ NOTICE NickServ :Sorry, it is too private here
```

**Comment by AlexDaniel** [comment]
This is because [``irc-to-me`` inlcudes ``irc-notice-me``](https://github.com/zoffixznet/perl6-IRC-Client/blob/master/docs/02-event-reference.md#event-map). It should not be like that. Not even sure what to do about it on our side.

**Comment by AlexDaniel** [comment]
This was fixed bcad26d746fa4b1e09299a562fcabac6a3866e25, closing in favor of #321.
